.travis.yml

```diff
@@ -15,13 +25,11 @@ matrix:
         - DISTRO=osx:xcode8.3
         - PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig:/usr/local/opt/zlib/lib/pkgconfig
 
-compiler:
-  - clang
-  - gcc
+  exclude:
+    - arch: ppc64le
+      env: 
+        - DISTRO=fedora:rawhide
```

Comment:
Why exclude ppc63le on rawhide ?

Reply:
I have exclude becuase i am getting error of "No matching manifest list entry" for the fedora:rawhide for ppc64le. Here is the issue link that has been already created: https://github.com/fedora-cloud/docker-brew-fedora/issues/76.

Thanks !!